### DISCUSSION TITLE
Help me Galaxy Brain

### DISCUSSION THREAD

**Discussion by shlee0882** [drknzz/GitHub-Achievements#120] [Q&A]
Thank you

**Comment by atlantus-network** [comment]
Galaxy Brain — resumo pratico:

- Precisa de **2 respostas aceitas** em Discussions Q&A.
- Repo tem que ser **publico**.
- Voce responde perguntas de **outras pessoas**.
- O autor marca sua resposta como **Answer**.

Dica: procure threads `is:unanswered` em repos com Discussions ativo. Respostas detalhadas tem mais chance de serem aceitas.

Se ajudou, marca como resposta aceita!
